bundle/config/mutator/override_compute_test.go

```diff
@@ -77,6 +89,31 @@ func TestOverrideDevelopmentEnv(t *testing.T) {
 	assert.Equal(t, "cluster2", bundle.Config.Resources.Jobs["job1"].Tasks[1].ExistingClusterId)
 }
 
+func TestOverridePipelineTask(t *testing.T) {
+	os.Setenv("DATABRICKS_CLUSTER_ID", "newClusterId")
```

Comment:
This must use `t.Setenv` to avoid tainting the environment for the remainder of the test run.

Reply:
Will include this in some followup.